src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -185,8 +191,24 @@ def generate_config(self):
             "peers": peers
         }, sorted(deps)
 
+    def get_active_daemon(self, daemon_descrs: List[DaemonDescription]) -> DaemonDescription:
```

Comment:
?
```suggestion

    @staticmethod
    def get_active_daemon(daemon_descrs: List[DaemonDescription]) -> DaemonDescription:
```

Reply:
-1 for `@staticmethod`. I need to get the active daemon for MDS and MGR to properly schedule daemon removals. And I need `self` to access the mgr. Adding `@staticmethod` force me to remove `@staticmethod` again later on.